prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudicator-notes/adjudicator-notes.component.ts

```diff
@@ -42,20 +49,31 @@ export class AdjudicatorNotesComponent implements OnInit {
 
   public onSubmit() {
     if (this.form.valid) {
```

Comment:
Not a change made by you, but you could use the `FormUtils.checkValidity()` that will if `form.valid` fails will automatically mark the form as touched.

```
if (this.formUtilsService.checkValidity(this.form)) {
  // ...
}
```

Reply:
and the addition of `Validators.required` with a `mat-error` would save a trip to the server and indicate their missing some information.